src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java

```diff
@@ -59,6 +93,11 @@ public MultiRangeParams withLabels(boolean withLabels) {
 
     if (withLabels) {
       params.add(Keyword.WITHLABELS.getRaw());
+    } else if (selectedLabels != null) {
```

Comment:
@sazzad16 perhaps you should reverse the order between withLabels and SELECTED_LABELS
This was SELECTED_LABELS will override withLabels

Reply:
@gkorland I followed the order from given doc https://github.com/RedisTimeSeries/JRedisTimeSeries/issues/63#issue-952778185

Reply:
I don't think there is a meaning for this order, the thing is that SELECTED_LABELS is more specific then WITHLABELS so I think in this case it should override.

Reply:
@gkorland done